validation/src/main/scala/hmda/validation/rules/lar/validity/V210.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.rules.lar.validity
+
+import java.text.SimpleDateFormat
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+
+object V210 extends EditCheck[LoanApplicationRegister] {
+  override def apply(lar: LoanApplicationRegister): Result = {
+    val date = lar.loan.applicationDate
+
+    try {
```

Comment:
Take a look at [this comment](https://github.com/cfpb/hmda-platform/pull/298#discussion_r61296170) that @jmarin made on an editCheck I wrote a few weeks ago. According to him, `try ... catch` isn't preferred in Scala; `Try() ... getOrElse` is more idiomatic. 

I played with this, and I have a working version that uses `Try` and `getOrElse` I'd be happy to share. Or if you'd enjoy working through it to make your own version, feel free.